packages/server/lib/plugins/child/run_plugins.js

```diff
@@ -114,6 +115,8 @@ class RunPlugins {
     Promise
     .try(() => {
       debug('Calling setupNodeEvents')
+      // setup all setters for 10.X+ non-supported config options
+      wrapNonMigratedOptions(initialConfig)
```

Comment:
Why are we wrapping non-migrated options?  I thought this was verified/errored in the DataContext.ProjectManger at this point when the project + testing type was loaded. Don't we want to verify the modifiedCfg returned by the setupNodeEvents func?

```js
import {
  validateNoBreakingConfig
  validateNoBreakingRootConfig
  validateNoBreakingTestingTypeConfig
} from '@packages/config'
...
.then((modifiedCfg) => {
  debug('plugins file successfully loaded')
  validateNoBreakingConfig(modifiedCfg, onWarning, onError)
  validateNoBreakingRootConfig(modifiedCfg, onWarning, onError)
  validateNoBreakingTestingTypeConfig(modifiedCfg, initialConfig.testingType, onWarning, onError)
```

Reply:
The reason is: pure UX.

If the error is thrown on the line where they set the wrong value, user only has to click on the link to open their file and fix the issue.

Reply:
what do you mean?

Reply:
I added a video to illustrate my point.